packages/browser/src/index.ts

```diff
@@ -25,4 +44,36 @@ export default class LogtoClient {
   constructor(logtoConfig: LogtoConfig) {
     this.logtoConfig = logtoConfig;
   }
+
+  public async signInWithBrowser(redirectUri: string) {
+    const { clientId, endpoint, requester, resources, scopes: originalScopes } = this.logtoConfig;
+
+    this.oidcConfig = await fetchOidcConfig(
+      `${endpoint}/oidc/.well-known/openid-configuration`,
```

Comment:
if we already used `URLSearchParams` then we can use [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) for safe join too

Reply:
Improved. PTAL @gao-sun 

```ts

const discoveryPath = '/oidc/.well-known/openid-configuration';

……

  public async signIn(redirectUri: string) {
    ……
    const discoveryEndpoint = new URL(discoveryPath, endpoint);
    this.oidcConfig = await fetchOidcConfig(discoveryEndpoint.toString(), requester);
    ……

```